src/components/hosts/HostsTable.tsx

```diff
@@ -51,16 +52,16 @@ const columns = [
 ];
 
 const hostToHostTableRow = (openRows: OpenRows) => (host: Host): IRow => {
-  const { id, status, role, createdAt, hardwareInfo = '' } = host;
-  const hwInfo = stringToJSON<Introspection>(hardwareInfo) || {};
-  const { cores, memory, disk } = getHostRowHardwareInfo(hwInfo);
+  const { id, status, role, createdAt, inventory: inventoryString = '' } = host;
+  const inventory = stringToJSON<Inventory>(inventoryString) || {};
+  const { cores, memory, disk } = getHostRowHardwareInfo(inventory);
 
   return [
     {
       // visible row
       isOpen: !!openRows[id],
       cells: [
-        id,
+        inventory.hostname || EMPTY,
```

Comment:
Sorry for not noticing it earlier, I think the EMPTY needs to be similar to EMPTY on hw columns?
```suggestion
        inventory.hostname || { title: DASH, sortableValue: '' },
```